WooCommerce/Classes/ViewRelated/Products/ProductSelector/ProductVariationSelectorViewModel.swift

```diff
@@ -51,7 +51,7 @@ final class ProductVariationSelectorViewModel: ObservableObject {
 
     /// All purchasable product variations for the product.
     ///
-    @Published private var productVariations: [ProductVariation] = []
+    @Published private(set) var productVariations: [ProductVariation] = []
```

Comment:
❓ Why do we need to make this change, do we need to read `productVariations` from outside?